tools/dnulist/program.cs

```diff
@@ -0,0 +1,241 @@
+        }
+    }    
+    
+    private static void CopyTarget(Project project, string target, string path, bool verbose = false)
+    {
+        const string packageRoot = "/Users/richlander/.dnx/packages";
+        string compilePath = Path.Combine(path, "compile");
+        string runtimePath = Path.Combine(path,"runtime");
+        
+        if (!Directory.Exists(path))
+        {
+            Directory.CreateDirectory(compilePath);
+            Directory.CreateDirectory(runtimePath);
+        }
+        else if (!Directory.Exists(compilePath))
+        {
+            Directory.CreateDirectory(compilePath);
+        }
+        else if (!Directory.Exists(runtimePath))
+        {
+            Directory.CreateDirectory(runtimePath);
+        }
+        
+        Dictionary<string,Package> packages;
+        var targetFound = project.Targets.TryGetValue(target, out packages);
+            
+        if (targetFound && packages.Count > 0)
+        {
+            foreach (var package in packages.Keys)
+            {   
+                Console.WriteLine("Copying package: {0}", package);
+                
+                var p = packages[package];         
+                if (p.Compile != null)
+                {
+                    foreach (var lib in p.Compile.Keys)
+                    {
+                        var source = Path.Combine(packageRoot,package,lib);
+                        var destination = Path.Combine(compilePath,Path.GetFileName(lib));;
+                        File.Copy(source,destination, true);
+                        
+                        if (verbose)
+                        {
+                            Console.WriteLine("**Compile: {0}", lib);
+                        }
+                    }
+                }
+                
+                if (p.Runtime != null)
+                {
+                    foreach (var lib in p.Runtime.Keys)
+                    {
+                        var source = Path.Combine(packageRoot,package,lib);
+                        var destination = Path.Combine(runtimePath,Path.GetFileName(lib));;
+                        File.Copy(source,destination,true);
+                        
+                        if (verbose)
+                        {
+                            Console.WriteLine("**Runtime: {0}", lib);
+                        }
+                    }
+                }
+            }
+        }
+        else if (targetFound)
+        {
+            Console.WriteLine("No packages found for {0}.",target);
+        }
+        else
+        {
+            Console.WriteLine("{0} is not a target for this project.", target);
+        }
+    }
+    
+    private static void PrintHelp()
+    {
+        Console.WriteLine("Primary arguments:");
+        Console.WriteLine("--target - prints targets");
+        Console.WriteLine("--target [target] - selects targets to print (the default) or otherwise act upon");
```

Comment:
I think you need both framework and runtime selection instead of all in one.  Compile time assets should only be selected from "RID"-less targets.  Runtime assets should be selected from the target with the RID.